automata/core/code_handling/py/reader.py

```diff
@@ -94,7 +94,7 @@ def _remove_docstrings(node: FSTNode) -> None:
                     if child_node is not node:
                         _remove_docstrings(child_node)
 
-        module = py_module_loader.fetch_module(module_dotpath)
+        module = cast(RedBaron, py_module_loader.fetch_module(module_dotpath))
```

Comment:
sorry for the ugly cast, will be going away in next PR

Reply:
nw